integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -21,5 +21,6 @@ fi
 pushd "$kubernetes_dir"
 ./init.sh
 bats nginx.bats
+bats busybox.bats
```

Comment:
The name of this file doesn't give any indication as to what the test is doing. I'd suggest renaming it to something like `k8s-uts+ipc-ns.bats`.